bin/cli.js

```diff
@@ -14,9 +14,18 @@ if (argv._.length < 3) {
   process.exit(1);
 }
 
+// Parse from/to as either a Regex or a string
+const isRegexMatch = /.*\/([gimy]*)$/;
```

Comment:
Slightly worried that this pattern might incorrectly identify for example paths as a regex. E.g. `/some-path/g` would match, but it could be a path string rather than a regex. Perhaps it'd be better to include a flag if you are passing in a regular expression as the `from` parameter, to be explicit and prevent accidental regex matches.

I'd propose the flag `isRegex` for this purpose.